### PR TITLE
[wasm][bindings] Add binding support for DataView

### PR DESCRIPTION
**Pull request by kjpou1**
Add bindings for DataView core object.

### CODE REVIEW

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by kjpou1** [comment]
@monojenkins build Linux WebAssembly

**Comment by kjpou1** [comment]
@monojenkins build failed

**Comment by kjpou1** [comment]
@monojenkins build failed

File: sdks/wasm/framework/src/WebAssembly.Bindings/Core/DataView.cs [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+		public void SetUint8 (int byteOffset, byte value, bool littleEndian = false) => Invoke ("setUint8", byteOffset, value, littleEndian);
+
+		private U UnBoxValue<U> (object jsValue) where U : struct
+		{
+			if (jsValue == null) {
+				throw new InvalidCastException ($"Unable to cast null to type {typeof (U)}.");
+			}
+
+			var type = jsValue.GetType ();
+			if (type.IsPrimitive) {
```
Comment (lewing) [inline sdks/wasm/framework/src/WebAssembly.Bindings/Core/DataView.cs]: is this ever going to be anything other than double in practice?

**Pull request merged**